samples/previews/azure-functions/service-bus/ASBFunctions_0/AzureFunctions.ASBTrigger.FunctionsHostBuilder/TriggerMessageHandler.cs

```diff
@@ -6,11 +6,19 @@
 
 public class TriggerMessageHandler : IHandleMessages<TriggerMessage>
 {
-    private static readonly ILog Log = LogManager.GetLogger<TriggerMessageHandler>();
+    static readonly ILog Log = LogManager.GetLogger<TriggerMessageHandler>();
+    readonly CustomComponent customComponent;
+
+    public TriggerMessageHandler(CustomComponent customComponent)
+    {
+        this.customComponent = customComponent;
+    }
 
     public Task Handle(TriggerMessage message, IMessageHandlerContext context)
     {
         Log.Warn($"Handling {nameof(TriggerMessage)} in {nameof(TriggerMessageHandler)}");
+        Log.Warn($"Custom component returned: {customComponent.GetValue()}");
```

Comment:
should we change the warn to info?

Reply:
This is to make the logs from the handlers stand out and be more prominent among the other info logs added by function.